backend/src/handlers/routes.py

```diff
@@ -285,30 +291,18 @@ async def create_route(req: Request, kml_file: UploadFile):
                 session.add(waypoint)
                 session.flush()
 
-        pos = 0
-
-        for stop_name, stop in stops.items():
-            stop_model = Stop(
-                name=stop_name, lat=stop.geometry.y, lon=stop.geometry.x, active=True
-            )
-            session.add(stop_model)
-            session.flush()
-
             routes_regex_pattern = r"<div>(.*?)(?:<br>)?<\/div>"
+            route_stops = re.findall(routes_regex_pattern, str(route.description))
```

Comment:
See BeautifulSoup for potentially better HTML parsing https://stackoverflow.com/questions/2136267/beautiful-soup-and-extracting-a-div-and-its-contents-by-id